include/dlaf/blas_tile.tpp

```diff
@@ -40,6 +40,55 @@ void gemm(blas::Op op_a, blas::Op op_b, T alpha, const Tile<const T, device>& a,
              c.ptr(), c.ld());
 }
 
+template <class T, Device device>
+void hemm(blas::Side side, blas::Uplo uplo, T alpha, const Tile<const T, device>& a,
```

Comment:
I have some concerns about not having `const` all parameters (i.e. `side`, `uplo`, `alpha`, `beta`)...maybe there is a reason not to make them `const`, but I don't see it.

This is valid for all functions in this file, so it is something that may be addressed (if needed) in another PR.

Reply:
I agree with you: having all parameters `const` looks safer. 
I have modified all the parameters in `blas_tile.tpp` , `lapack_tile.tpp` and related `.h` and it works well. I can move these changes into a new PR if you agree.